addons/account_edi_ubl_bis3/models/account_edi_format.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+from odoo.exceptions import UserError
+from odoo.tests.common import Form
+
+
+COUNTRY_EAS = {
+    'HU': 9910,
+
+    'AD': 9922,
+    'AL': 9923,
+    'BA': 9924,
+    'BE': 9925,
+    'BG': 9926,
+    'CH': 9927,
+    'CY': 9928,
+    'CZ': 9929,
+    'DE': 9930,
+    'EE': 9931,
+    'UK': 9932,
+    'GR': 9933,
+    'HR': 9934,
+    'IE': 9935,
+    'LI': 9936,
+    'LT': 9937,
+    'LU': 9938,
+    'LV': 9939,
+    'MC': 9940,
+    'ME': 9941,
+    'MK': 9942,
+    'MT': 9943,
+    'NL': 9944,
+    'PL': 9945,
+    'PT': 9946,
+    'RO': 9947,
+    'RS': 9948,
+    'SI': 9949,
+    'SK': 9950,
+    'SM': 9951,
+    'TR': 9952,
+    'VA': 9953,
+
```

Comment:
why the empty lines?

Reply:
For formating. The codes are incremental, but there are holes. This is easier to read, and will be easier to add codes when we will support them or the holes are filled by the standard.